Model/LinkOrgIdentityEnroller.php

```diff
@@ -516,18 +527,35 @@ public function createOrgIdentity($registered_user, $cmp_attibutes_list, $email_
         )
       );
     }
-  
-    // The Subject DN is fetched as the attribute distinguishedName
-    if(!empty($cmp_attibutes_list['distinguishedName'])){
+
+    // Certificate handle
+    if(CertUtils::consumeDecideVoPersonCertAttr($plg_cfg, $cmp_attibutes_list)) {
+      list($subject_value, $issuer_value) = CertUtils::getEnvValuesFromCache($plg_cfg, $cmp_attibutes_list);
+      if(!empty($subject_value)) {
+        $association_data['Cert'] = array();
+        $sdn_values = explode(';', $subject_value);
+        foreach($sdn_values as $dn) {
+          $association_data['Cert'][] = array(
+            'subject' => $dn,
+            'type' => CertEnum::X509,
+            'actor_identifier' => $cmp_attibutes_list[$user_id_attribute],
+          );
+        }
+      }
+    } else {
+      list($subject_value, $issuer_value) = CertUtils::getEnvValuesFromCache($plg_cfg, $cmp_attibutes_list);
```

Comment:
This can be put outside of if-else condition?

Reply:
ok